### PR TITLE
Add DISABLE_AUTO_IMPORT_DOC_BLOCK_NAMES option

### PR DESCRIPTION
**Pull request by alexndlm**

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Hi, thanks for the PR.
Could you move to `importNames()` method so we have it together? The similar way the parallel() method is.

**Comment by alexndlm** [comment]
Cool. Done.

File: packages/Config/RectorConfig.php [hunk 1]
```diff
@@ -69,22 +69,12 @@ public function skip(array $criteria): void
         $parameters->set(Option::SKIP, $criteria);
     }
 
-    public function importNames(): void
+    public function importNames(bool $disableImportDocBlockNames = false): void
```
Comment (TomasVotruba) [inline packages/Config/RectorConfig.php]: The value should be inverted, to avoid double neagive mix.


```diff
-disableImport = false
+import = true
```

Comment (alexndlm) [reply packages/Config/RectorConfig.php]: I did a bit more refactoring to make the `importNames` method call more clear and not confusing.

Comment (alexndlm) [reply packages/Config/RectorConfig.php]: @TomasVotruba Check please this PR https://github.com/rectorphp/rector-nette/pull/76

Comment (alexndlm) [reply packages/Config/RectorConfig.php]: CI green again 🌲

**Review by TomasVotruba — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thank you 😊

**Comment by alexndlm** [comment]
@TomasVotruba Would you please make new Rector release?

We need this feature to use latest Rector in our projects.

**Comment by TomasVotruba** [comment]
Next version is comming up next week along with few other fixes.
For time being, you can use lock to commit in `composer.json`.

**Comment by alexndlm** [comment]
Ok, thanks :)

**Pull request merged**